source/client-backpressure/client-backpressure.md

```diff
@@ -40,6 +40,9 @@ the connection and request rate limiters to prevent and mitigate overloading the
 An error is considered retryable if it includes the "RetryableError" label. This error label indicates that an operation
 is safely retryable regardless of the type of operation, its metadata, or any of its arguments.
 
+Note that for the initial draft of the spec, only errors that have both the RetryableError label and the
+SystemOverloadedError label are eligible for the retry backoff loop.
```

Comment:
What is "initial draft of the spec"? As far as I understand, once the changed proposed in this PR are approved and merged, they become part of the driver specifications, and not "initial draft".

Reply:
removed